### PR TITLE
add links to related repos

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,34 @@
 - [Joyent Ui Toolkit](https://joyent-ui-toolkit.now.sh/)
 - [Create Instance Prototype](https://create-instance-prototype.now.sh)
 
+## Related
+
+#### Copilot
```
Comment (yaili) [inline README.md]: Is it CoPilot?

File: README.md [hunk 2]
```diff
@@ -21,6 +21,34 @@
...
+#### Tooling
+
+- [stylelint-config-joyent-portal](https://github.com/yldio/stylelint-config-joyent-portal)
+- [joyent-react-scripts](https://github.com/yldio/joyent-react-scripts)
+- [babel-preset-joyent-portal](https://github.com/yldio/babel-preset-joyent-portal)
+- [eslint-config-joyent-portal](https://github.com/yldio/eslint-config-joyent-portal)
+
 ## Contribute
 
 See [the contribute file](CONTRIBUTING.md)!
```
Comment (yaili) [inline README.md]: Is there an extra space before the exclamation mark?

File: README.md (hunk 1 above)
Comment (ramitos) [reply README.md]: I'm never sure, but I think you're right 🙂

File: README.md (hunk 2 above)
Comment (ramitos) [reply README.md]: I thought so too, but it isn't

**Pull request merged**